### PR TITLE
feat(plugin-input-exercise): Add feedback placeholder text

### LINKED ISSUE #2976: Add placeholder text for feedback field
In context of input-exercise plugin.

### PR DESCRIPTION
**Pull request by AsterMiha**

### CODE REVIEW

File: src/serlo-editor/plugins/input-exercise/index.ts [hunk 1]
```diff
@@ -33,22 +32,20 @@ export type InputExercisePluginState = ReturnType<
...
-  feedback?: ChildStateTypeConfig
+  feedback: ChildStateTypeConfig
 }
 
 export type InputExerciseProps = EditorPluginProps<
   InputExercisePluginState,
   InputExerciseConfig
 >
 
-const defaultFeedback = { plugin: EditorPluginType.Text }
```
Comment (elbotho) [inline src/serlo-editor/plugins/input-exercise/index.ts:44]: I'd keep the default so other integrations do not have to provide something here (thinking of edusharing)

**Review by elbotho — CHANGES_REQUESTED** [review]
Thanks!
See comment for one change wish.

File: src/serlo-editor/plugins/input-exercise/index.ts (hunk 1 above)
Comment (AsterMiha) [reply src/serlo-editor/plugins/input-exercise/index.ts:44]: Brought it back!

**Review by elbotho — APPROVED** [review]
Nice, thank you!

**Pull request merged**